README.adoc

```diff
@@ -1,17 +1,22 @@
-// Do not edit this file (e.g. go instead to src/main/asciidoc)
+////
+DO NOT EDIT THIS FILE. IT WAS GENERATED.
+Manual changes to this file will be lost when it is generated again.
+Edit the files in the src/main/asciidoc/ directory instead.
+////
+
 
 image:https://img.shields.io/travis/spring-cloud/spring-cloud-gcp.svg?branch=master&label=master["Master Build Status", link="https://travis-ci.org/spring-cloud/spring-cloud-gcp"]
 image:https://img.shields.io/travis/spring-cloud/spring-cloud-gcp.svg?branch=1.1.x&label=1.1.x["1.1.x Build Status", link="https://travis-ci.org/spring-cloud/spring-cloud-gcp"]
 image:https://img.shields.io/travis/spring-cloud/spring-cloud-gcp.svg?branch=1.0.x&label=1.0.x["1.0.x Build Status", link="https://travis-ci.org/spring-cloud/spring-cloud-gcp"]
 
 = Spring Framework on Google Cloud Platform
 
-The Spring Cloud GCP project makes it easy for Spring users to run their applications on Google Cloud Platform.
-You can check our https://github.com/spring-cloud/spring-cloud-gcp/blob/master/README.adoc[project website] for more information.
+This project makes it easy for Spring users to run their applications on Google Cloud Platform.
+You can check our project website https://cloud.spring.io/spring-cloud-gcp[here].
 
-For a deep dive into the project, refer to the https://cloud.spring.io/spring-cloud-static/spring-cloud-gcp/1.2.2.RELEASE/reference/html/[Spring Cloud GCP 1.2 Reference Document] or the https://googleapis.dev/java/spring-cloud-gcp/latest/index.html[latest Javadocs].
+For a deep dive into the project, refer to the https://cloud.spring.io/spring-cloud-static/spring-cloud-gcp/1.2.0.RELEASE/reference/html/[Spring Cloud GCP 1.2 Reference Document].
```

Comment:
This change to an older version doesn't seem right.
We probably need to update the source from which this is generated.

Reply:
The new docs will be under https://docs.spring.io/spring-cloud-gcp/docs/1.2.2.RELEASE/ e.g. for 1.2.2.RELEASE version.

Reply:
Hey @marcingrzejszczak, were the changes you made to this file manually made or did you generate the changes from sources somewhere?

I think we were just wondering why you changed the version from `1.2.3.RELEASE` to `1.2.0.RELEASE`.

Reply:
Ah I see where the changes are coming from... I'll take care of merging this.